### ISSUE TITLE
Add possibility to modify metadata in tus.Upload options

### ISSUE THREAD

**Issue by gleam-ru** [safrazik/vue-file-agent#58]
![image](https://user-images.githubusercontent.com/6701480/79807095-9fe02a00-8372-11ea-83a6-a7ddbba8a0f1.png)

I have to modify metadata block in tus initialization (screenshot above). But it is impossible.
tus-js-client allows to user-define it.

Could you implement it in your package? :)

**Comment by safrazik** [comment]
Thanks for the issue. Will be available in next version. Hope you can make use of a workaround until then.

**Comment by safrazik** [comment]
Available from Version `1.7.0-beta.0`
`:resumable` prop accepts boolean or a callback that returns an options object for tus js client.

```html
<VueFileAgent
  :resumable="resumableOptions"
></VueFileAgent>
```

```javascript
export default {
  // ...
  data: {
    // ...
    resumableOptions: (fileRecord) => {
      const file = fileRecord.file;
      return {
        // both retryDelays and metadata are optional
        retryDelays: [0, 3000, 5000, 10000, 20000],
        metadata: {
            filename: file.name,
            filetype: file.type
        },
      }
    }
    // ...
  }
  // ...
}
```
